azure-pipelines-release.yml

```diff
@@ -0,0 +1,42 @@
+trigger:
+  branches:
+    include:
+    - master
+
+parameters:
+- name: SignTypeSelection
+  displayName: Sign type
+  type: string
+  default: Real
+  values: [ 'Test', 'Real' ]
+
+  variables:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  BuildConfiguration: Release
+  BuildPlatform: Any CPU
+  NUGET_PACKAGES: $(Agent.TempDirectory)/.nuget/packages
+  SigningType: ${{ parameters.SignTypeSelection }}
+
+jobs:
+- job: Windows
+  pool: VSEng-MicroBuildVS2019
+  steps:
+ - template: azure-pipelines-common-windows.yml
+ - task: PowerShell@1
+    condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
+    displayName: 'Push VSIX'
+    inputs:
+      scriptType: inlineScript
+      inlineScript: |
+        # Setting these environment variables because they are used in the script for the upload endpoint: 
+     
+        $env:APPVEYOR_REPO_PROVIDER = "github"
+        $env:APPVEYOR_REPO_NAME = "microsoft/dotnet-apiport"
```

Comment:
These env vars aren't used now, are they?
Also it is the only script in a task labeled "Push VSIX", but it doesn't push any VSIX any more. Was it supposed to invoke the `vsix.ps1` script?
I recommend you drop the condition since this pipeline should do its job (when invoked) regardless of what the branch is called. So I think you can replace this whole task with this:

```yaml
  - powershell: tools/appveyor/vsix.ps1
    displayName: Push VSIX
```

Reply:
Accidently get the wrong code here. Actually, it makes sense to have vsix.ps1 does the push as well. I will update a commit later.

Reply:
Commited a fix